### DISCUSSION TITLE
Is there a stable version?

### DISCUSSION THREAD

**Discussion by lcfd** [NeogitOrg/neogit#390] [Q&A] [answered]
Hi! 👋 

I need that Neogit works without any flaw caused by changes in the repository.
With Packer, if do a Sync it will download the latest commit and I don't know how safe it is.

Is there a stable version?
Is there a way to set this stable version in Packer?

Thank you very much 😄

**Comment by ttytm** [comment, marked as answer]
Hey @icfd

Neogit is in pretty early development. But you can instruct packer to use a specific commit to protect yourself from new braking changes. E.g.,

```lua
use { "TimUntersberger/neogit", commit = "4cc4476acbbc772f29fd6c1ccee43f58a29a1b13" },
```

**Reply by lcfd** [reply]
Hi @tobealive 
(I think you tagged the wrong person 🙈)

Thank you for the answer!

Is there a commit that you would suggest as stable or doesn't make sense?

**Reply by ttytm** [reply]
Oh noes 🙈 it looks like I did, sorry.

I'm using the commit I've posted. After that one things somehow got better and worse at the same time 😅. 

Until some fixes are implemented I'll stick with it. I can leave an update when I think it's ripe to move on. 
Personal opinions here though.

**Reply by ttytm** [reply]
Looks like we are good to go ☺
